client/sidebar/footer/SidebarFooter.tsx

```diff
@@ -18,16 +20,24 @@ const SidebarFooter = (): ReactElement => {
 		}
 	`;
 
+	const state = useIsCallEnabled();
+
+	if (state) {
+		return <VoiceFooter />;
+	}
+
 	return (
-		<Box
-			is='footer'
-			pb='x12'
-			pi='x16'
-			height='x48'
-			width='auto'
-			className={sidebarFooterStyle}
-			dangerouslySetInnerHTML={{ __html: String(settings.get('Layout_Sidenav_Footer')).trim() }}
-		/>
+		<Footer>
+			<Box
+				is='footer'
+				pb='x12'
+				pi='x16'
+				height='x48'
+				width='auto'
+				className={sidebarFooterStyle}
+				dangerouslySetInnerHTML={{ __html: String(settings.get('Layout_Sidenav_Footer')).trim() }}
```

Comment:
There is no safe method to put content here?

Reply:
This is the way that was being done before, you can check the file history, we didn't alter anything there